website/api/controllers/entrance/signup.js

```diff
@@ -115,6 +131,43 @@ the account verification message.)`,
       });
     }
 
+    // If "Try Fleet Sandbox" was provided as the signupReason, this is a user signing up to try Fleet Sandbox.
+    if(signupReason === 'Try Fleet Sandbox') {
+
+      // Creating an expiration JS timestamp for the Fleet sandbox instance. NOTE: We send this value to the cloud provisioner API as an ISO 8601 string.
+      let fleetSandboxExpiresAt = Date.now() + (24*60*60*1000);
+
+      // Send a POST request to the cloud provisioner API
+      let cloudProvisionerResponse = await sails.helpers.http.post('https://sandbox.fleetdm.com/new', {
+        'name': firstName + ' ' + lastName,
+        'email': emailAddress,
+        'password': newUserRecord.password, //« Sending the hashed password to the Fleet Sandbox instance
+        'sandbox_expiration': new Date(fleetSandboxExpiresAt).toISOString(), // sending expiration_timestamp as an ISO string.
+      })
+      .timeout(5000)
+      .intercept('non200Response', 'couldNotProvisionSandbox');
```

Comment:
I'm also not seeing the shared secret here, or the strategy involving the `demoKey` as specced at the bottom of https://docs.google.com/document/d/1DUR6XtsuxTrXIrpkRR5J1KFfADwL9CrcSRyR2joozZg/edit

Reply:
The shared secret has been added to the cloud provisioner request as a header using an environment variable named `cloudProvisionerSecret`